packages/components/src/schemas/main.ts

```diff
@@ -9,3 +10,21 @@ export const schema: TSchema = {
   ...IsomerPageSchema,
   ...componentSchemaDefinitions,
 }
+
+export const siteSchema: TSchema = {
+  $schema: "http://json-schema.org/draft-07/schema#",
```

Comment:
curious: what's this ah?

Reply:
This is a marker on JSON schemas to indicate which specifications the JSON schema conforms to